packages/block-library/src/heading/edit.js

```diff
@@ -62,19 +61,11 @@ function HeadingEdit( {
 		<>
 			<BlockControls>
 				<HeadingToolbar align={ align } setAttributes={ setAttributes } minLevel={ 2 } maxLevel={ 5 } selectedLevel={ level } onChange={ ( newLevel ) => setAttributes( { level: newLevel } ) } />
+				<AlignmentToolbar value={ align } onChange={ ( nextAlign ) => {
+					setAttributes( { align: nextAlign } );
+				} } />
 			</BlockControls>
 			<InspectorControls>
-				<PanelBody title={ __( 'Heading Settings' ) }>
-					<p>{ __( 'Level' ) }</p>
-					<HeadingToolbar minLevel={ 1 } maxLevel={ 7 } selectedLevel={ level } onChange={ ( newLevel ) => setAttributes( { level: newLevel } ) } />
```

Comment:
I think we shouldn't remove the `HeadingToolbar`  from the sidebar because it allows H1 while the block toolbar's one doesn't. (That's done on purpose to deemphasize that heading level.

Reply:
There is also related PR #14889 which adds heading hierarchy checker which will require having those levels in the sidebar to make it easier to fix the issues reported.

Reply:
Oh I see, I'll put them back, wasn't aware of all this, just figured they are duplicated controls. However even with all the explanation, it still feels duplicate.